index.html

```diff
@@ -3885,6 +3885,29 @@ <h3>JSON-LD</h3>
 base media type, `application/vc+ld+json`.
         </p>
 
+        <p>
+As elaborated upon in Section <a href="#static-processing"></a>, some software
+applications might not perform dynamic JSON-LD processing, which uses features
+such as JSON-LD compaction and expansion. Static processors can elect to reject
+unrecognized URL values in the `@context` property, for example. Authors of
+<a>conforming documents</a> are also advised that use of certain dynamic
+JSON-LD processor features might reduce interoperability. These features
+include the following:
+        </p>
+
+        <ul>
+          <li>
+Use of JSON-LD keywords in the `@context` value that globally modify document
+value processing, such as global settings of `@base` and `@language`
+          </li>
+          <li>
+Use of full URLs for JSON-LD terms and types, instead of the compact forms of
+those values as are defined in application-specific JSON-LD context files
```

Comment:
What about
"instead of the compact forms of those values whether or not they are defined in application-specific JSON-LD context files"
This covers the cases when a new context file is created or when a new context file is not created.

Reply:
There *is* no "compact form" of such URL values, if they're not mapped (I think "defined" remains the wrong verb for what `@context` files do, whether I accidentally re-introduced it or overlooked it, in a relevant comment or suggestion related to this thread) by an `@context` file, no matter whether that file is old, new, generic, application-specific, etc.

The thing called "compact form" here — probably better to be called the "unmapped form" — is the naked word, the non-URL. The full URL is what that "unmapped form" is mapped to by *some* `@context` identified by the JSON-LD document in question, which is probably one of the documents cited in the VC specifications, unless the "unmapped form" in question is not mapped by any of those VC `@context` files.

These "unmapped forms" *might* be validly referred to as "JSON keys" given that a JSON file contains some number of key/value pairs; and a JSON-LD file contains some number of key/value pairs where the JSON-LD-relevant keys are mapped by a file identified by the `@context` property, and some additional number of key/value pairs involving keys that remain un-mapped by any identified `@context` are ignored as non-conforming or uncrecognized.

```suggestion
those values as are mapped by application-specific JSON-LD context files
```

Beyond that, I don't believe I drafted these bullets nor their introductory paragraph, though I did content-edit both, so I'm hoping earlier author(s) will chime in on whether they see their meaning as being preserved, lost, clarified, distorted, etc., by current efforts.